aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -130,9 +151,9 @@ The generated HTML will include the areas segment and will be as follows:
 > [!TIP]
 > For MVC Areas to work in your web application, the route template must include a reference to the area if it exists. That template (which is the second parameter of the ```routes.MapRoute``` method call) will look as follows: ```template: "{area:exists}/{controller=Home}/{action=Index}"```.
 
+- - -
 
-
-## asp-protocol
+### asp-protocol
 
```

Comment:
... protocol (such as ```https```) in ...

to

... protocol, such as `https`, in ...

... will look as follows.

to

... will appear as: